ui/component/autoplayCountdown/view.jsx

```diff
@@ -155,7 +160,11 @@ function AutoplayCountdown(props: Props) {
               iconRight={skipPaid ? ICONS.WALLET : ICONS.REPLAY}
               onClick={() => {
                 setTimerCanceled(true);
-                doReplay();
+                if (skipPaid) {
```

Comment:
- [ ] Autoplay next is broken -- it just replays itself

Steps:
- http://localhost:9090/@BrittanyVenti:0/they-banned-me-for-being-too-funky-fresh:a in incognito.  Autoplay-next should be on by default
- Drag towards the end and let it continue to the next video.  When the countdown is over, it replays itself rather than the next video.

Reply:
Fixed ?

Reply:
Just tried on salt.  Still happening.

Reply:
And now?